tests/conftest.py

```diff
@@ -65,6 +65,7 @@
 from aea.crypto.fetchai import FetchAICrypto
 from aea.identity.base import Identity
 from aea.mail.base import Address
+from aea.test_tools.click_testing import CliRunner  # noqa
```

Comment:
Could also do an assignment

CliRunner = ImportedCliRunner

Then you don;t need to ignore